### DISCUSSION TITLE
find file when switching to the root of a drive I get all kinds of entries which are not folders or files

### DISCUSSION THREAD

**Discussion by maneum1** [minad/vertico#431] [Q&A] [answered]
Hello, 

using find file when switching to the root of a drive I get all kinds of entries which are not folders or files
see attached images.
The additional entries look like protocol or command names.
e.g.
su:
ftp:

What settings are required to show only files and folders, ... stuff that would normally show in a dired buffer?

![image](https://github.com/minad/vertico/assets/22628333/bc9c9de8-3574-460b-a9e5-60131f339623)

**Comment by maneum1** [comment]
forgot to mention, I'm on windows 10, using emacs 29.1_2

**Comment by minad** [comment, marked as answer]
Yes, these protocols are supported by Tramp in order to allow you to access remote files. If you enter `f:/some/file/path` you should only see files though.

**Reply by maneum1** [reply]
When I enter `f:/`
then I see all the protocols,  only after switching to a subfolder `f:/cpp/` they go away
would be nice if they only show while I've entered `f:`
or to have an option to manage the protocols.

**Reply by minad** [reply]
It could be that this issue has been fixed in Emacs 29.2, see https://github.com/minad/vertico?tab=readme-ov-file#tramp-hostname-and-username-completion. In any case, this is not a Vertico feature, but a feature of the `find-file` completion table.

**Reply by maneum1** [reply]
Thanks for explaining, doing some research I now understand a bit better whats going on behind the scenes. But not yet enough to eliminate the tramp offers.  Hopefully 29.2 improves this in the future.
